app/assets/stylesheets/admin.scss

```diff
@@ -0,0 +1,103 @@
+@charset "utf-8";
+
+@import "normalize-rails";
+@import "variables";
+@import "flashes";
+
+header {
+  display: flex;
+  background: $hotline-bling-pink;
+  padding: 0 1em;
+  margin-bottom: 2em;
+}
+
+main {
+  font-size: 18px;
+  margin: 0 auto;
+  max-width: 80%;
```

Comment:
What about these as variables, also? I'm thinking about all these mystery guests and wanting to refactor the CSS at some point soon. I'm worried that trying to untangle all of the unlabeled values will be a pain 😕

Reply:
Sure, I'll factor this out. I chose this so we'd have a central column of some size with padding on the sides. Changing it won't break anything (but that could be clearer).